Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:16.14.0
+WORKDIR /chase-car-dashboard
+COPY . .
```

Comment:
Do you really want to copy the whole filesystem into the docker image? This means any changes in the filesystem will require the docker image to be relaunched.

If this is intended ignore this. If not consider adding the `-v` flag to your `docker run` command to mount the source code in a shared volume so that changes are mirrored between docker and the host computer

Reply:
We thought copying the whole filesystem would be necessary, but we are not 100% sure what volumes actually do. We also want the package to change every time someone pushes to main, and not when the run command is triggered through shell script. Maybe we can talk more about this option, but Sarthak and I are just a little unfamiliar and believed that copying the whole filesystem would be necessary.